src/routes/chains/entities/backbone.entity.ts

```diff
@@ -13,7 +13,7 @@ export class Backbone implements DomainBackbone {
   @ApiProperty()
   secure: boolean;
   @ApiProperty()
-  settings: object;
+  settings: Record<string, string>;
```

Comment:
Why are two backbone entities?

Reply:
One is what we consider the domain entity that can be used by any route: `src/domain/backbone/entities/backbone.entity.ts`

The other one is what the route returns including the OpenAPI documentation for it: `src/routes/chains/entities/backbone.entity.ts`